src/lib.rs

```diff
@@ -16,7 +16,6 @@
 #![cfg_attr(feature = "nightly", feature(test))]
 #![cfg_attr(feature = "swap", feature(allocator_api))]
 #![cfg_attr(feature = "swap", feature(slice_ptr_get))]
-#![cfg_attr(feature = "swap", feature(nonnull_slice_from_raw_parts))]
```

Comment:
Keep it for now, older nightly compiler (like the one TiKV is using) hasn't opt-in this feature yet.